.github/workflows/build.yml

```
@@ -55,7 +55,7 @@ jobs:
         mkdir -p .build/msys64/dev/shm
         mkdir -p .build/msys64/dev/mqueue
         cp -rf src/* .build/msys64/
-        sed -i '1i#!/usr/bin/env python3\' .build/msys64/mingw64/bin/qmk-script.py
+        sed -i '1i#!/usr/bin/env python3 -Xutf8\' .build/msys64/mingw64/bin/qmk-script.py
```

```suggestion
        sed -i '1i#!/usr/bin/env -S python3 -Xutf8\' .build/msys64/mingw64/bin/qmk-script.py
```
Although, setting this in `bin/qmk` instead would work for normal MSYS2 as well.

---

Had initially tested but doesnt work, throws an error (will add when i am at my machine)

---

Going to merge this as is, so its in the next release. Feel free to raise an issue if you think it needs to be looked into more.